package.json

```
@@ -49,7 +49,7 @@
   },
   "homepage": "https://github.com/Financial-Times/n-eventpromo",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 12.0.0"
```

```suggestion
    "node": "12.x"
```

To prevent it running on v13, which does not have LTS.